requirements.txt

```diff
@@ -1,3 +1,3 @@
-tsfresh>=0.13.0
+tsfresh>=0.15.1
```

Comment:
I think 0.15.0 is fine for the lower limit.  The only change between 0.15.0 and 0.15.1 is a bugfix for tsfresh documentation